src/components/member/tabs/contract-tab/InsuranceMandate.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { ButtonLink } from '../../../../../shared/hedvig-ui/button'
+import { Card } from '../../../../../shared/hedvig-ui/card'
+import { Contract } from '../../../../api/generated/graphql'
```

Comment:
You can fix these auto-imports by setting so `Imports relative to tsconfig` or similar in your Typescript preferences. Also setting `src` and `shared` as `Resource Root`
```suggestion
import { ButtonLink } from 'shared/hedvig-ui/button'
import { Card } from 'shared/hedvig-ui/card'
import { Contract } from 'api/generated/graphql'
```